sklearn/utils/multiclass.py

```diff
@@ -174,7 +174,16 @@ def check_classification_targets(y):
 
 
 def type_of_target(y):
-    """Determine the type of data indicated by target `y`
+    """Determine the type of data indicated by target ``y``
```

Comment:
From the numpydoc, it seems that the one-line summary should not contain any variable names (and should finish with a full stop).